ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss

```diff
@@ -0,0 +1,389 @@
+@import '../../../styles/variables.scss';
+
+$container-width: 1200px;
+
+.main > * {
+    width: 100%;
+}
+
+section {
+    box-sizing: border-box;
+    text-align: center;
+
+    &:nth-of-type(even) {
+        background-color: #F6F7FA;
+    }
+
+    &.introduction {
+        max-width: 1440px;
+        padding-top: 108px;
+        padding-bottom: 18px;
+        text-align: center;
+        background-image: url(/assets/images/hero.png);
+        background-repeat: no-repeat;
+        background-position: bottom;
+        background-size: contain;
+
+        .title {
+            margin-top: 0;
+            margin-bottom: 16px;
+        }
+
+        .logos {
+            img {
+                width: 124px;
+                height: 32px;
```

Comment:
most of resources recommend to set the image size in img html attributes 
https://html.com/attributes/img-width/
https://stackoverflow.com/questions/1247685/should-i-specify-height-and-width-attributes-for-my-imgs-in-html

Reply:
Added at 87ba2de5e05f0a11bacb10d5f1856af3f0db3772

Reply:
do we want to fix it here too
https://github.com/broadinstitute/ddp-angular/pull/631/files#diff-cafcef202ba08651416bc2f80b6ec9250e0389f0bfd9098fb846c24e3671111cR108

Reply:
Fixed at 21980104ef6909da0ed1e483d50c48b3b9c42f58